components/eam/bld/build-namelist

```diff
@@ -3903,6 +3903,9 @@ if ($chem =~ /_mam/) {
 #Sea Salt aerosol emission scale factors
 add_default($nl, 'seasalt_emis_scale');
 
+#DMS emission scale factor
+add_default($nl, 'dms_emis_scale');
```

Comment:
@mahf708 , can you please change line 3907 above to
`add_default($nl, 'dms_emis_scale') if (get_default_value('dms_emis_scale')) ;`

Default value is only set when phys="default" (or via user_nl_eam). When none is set, say for FIDEAL, it would complain no default is set when calling add_default.